extension-stack-trace/src/main/java/ai/h2o/extensions/stacktrace/StackTraceExtension.java

```diff
@@ -0,0 +1,104 @@
+package ai.h2o.extensions.stacktrace;
+
+import water.AbstractH2OExtension;
+import water.H2O;
+import water.util.Log;
+
+import java.util.Date;
+import java.util.Map;
+import java.util.Set;
+
+public class StackTraceExtension extends AbstractH2OExtension {
+  private int interval = 10; // 10 seconds
```

Comment:
Very aggressive interval - maybe we can be more conservative? It will generate a lot of output in the logs

Reply:
Idea: do not use any default - force the user to set a reasonable value that works for them

Reply:
I would still prefer to set some default value, again to minimize the configuration, but agree that it should be higher.